src/test/java/com/github/pagehelper/util/ExecutorUtilTest.java

```diff
@@ -0,0 +1,64 @@
+package com.github.pagehelper.util;
+
+import static org.mockito.AdditionalMatchers.or;
+import static org.mockito.Matchers.anyBoolean;
+import static org.mockito.Matchers.isA;
+import static org.mockito.Matchers.isNull;
+
+import com.diffblue.deeptestutils.Reflector;
+import com.github.pagehelper.util.ExecutorUtil;
+import org.apache.ibatis.cache.CacheKey;
+import org.apache.ibatis.executor.Executor;
+import org.apache.ibatis.mapping.BoundSql;
+import org.apache.ibatis.mapping.MappedStatement;
+import org.apache.ibatis.session.Configuration;
+import org.apache.ibatis.session.ResultHandler;
+import org.apache.ibatis.session.RowBounds;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import java.lang.reflect.Array;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.sql.SQLException;
+import java.util.ArrayList;
+
+@PowerMockIgnore({"javax.management.*"})
+@RunWith(PowerMockRunner.class)
+public class ExecutorUtilTest {
+
+  @Rule public ExpectedException thrown = ExpectedException.none();
+
+  /* testedClasses: ExecutorUtil */
+
+  /*
+   * Test generated by Diffblue Deeptest.
+   * This test case covers the entire method.
+   */
+  @PrepareForTest({Configuration.class})
+  @Test
+  public void getExistedMappedStatementInputNotNullNotNullOutputNotNull()
```

Comment:
The already have a test that covers this line according to the online demo. Even though there is a single line missing, I think this is down to bluecov.

Reply:
Agreed. Removed.